homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -434,35 +434,33 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
         if DATA_KEY not in hass.data:
```

Comment:
On the line above, please invert the check and return early if true. Then we can outdent below:
```py
if config_entry.data[CONF_FLOW_TYPE] != CONF_DEVICE:
    return

...
```